pkg/inject/sidecar_test.go

```diff
@@ -158,6 +158,36 @@ func TestInjectSidecarWithEnvVarsOverridePropagation(t *testing.T) {
 	assert.Contains(t, dep.Spec.Template.Spec.Containers[0].Env, corev1.EnvVar{Name: envVarServiceName, Value: "testapp.default"})
 }
 
+func TestInjectSidecarWithVolumeMounts(t *testing.T) {
+	// prepare
```

Comment:
Could you please add an explicit test for the condition I mentioned in the previous comment? Add a volume/mount at the root of the spec and make sure that it does *not* get mounted.

Reply:
Done